### PR TITLE
Factor out Prefix helper functions from go-ipfs/merkledag

### PR DESCRIPTION
**Pull request by kevina**

### CODE REVIEW

File: prefix.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+// V0CidPrefix returns a prefix for CIDv0
+func V0CidPrefix() Prefix { return v0CidPrefix }
+
+// V1CidPrefix returns a prefix for CIDv1 with the default settings
+func V1CidPrefix() Prefix { return v1CidPrefix }
+
+// PrefixForCidVersion returns the Protobuf prefix for a given CID version
+func PrefixForCidVersion(version int) (Prefix, error) {
```
Comment (Stebalien) [inline prefix.go]: IMO, this is *way* too protobuf specific to go in go-cid.

**Comment by kevina** [comment]
> Do the NewPrefixV0 and NewPrefixV1 functions not work for you?

They would if they existed, but I only fond NewCidV0 and NewCidV1.

File: prefix.go (hunk 1 above)
Comment (kevina) [reply prefix.go]: Fair enough.  Will change to make them more general.

**Comment by Stebalien** [comment]
> They would if they existed, but I only fond NewCidV0 and NewCidV1.

They're in cid.go.

**Comment by kevina** [comment]
> They're in cid.go.

Oops.  Forgot to update my local git copy.  Sorry about that.

Closing.

**Comment by Stebalien** [comment]
Np. I've done that countless times.

**Pull request closed without merging**